pyproject.toml

```diff
@@ -20,9 +20,13 @@ python = "^3.6"
 typing-extensions = ">=3.0.0"
 nr-io-lexer = "^1.0.0"
 dataclasses = { version = "^0.6", python = "<3.7" }
+astor = { version = "^0.8.1", optional = true }
 
 [tool.poetry.extras]
-astor = ['astor ^0.8.1']
+astor = ["astor"]
+
+[tool.poetry.scripts]
+builddsk = "builddsl.wrapper.__main__:main"
```

Comment:
Where.. does this come from? `builddsl.wrapper` does not exist

Reply:
this is actually not needed